pkg/test/runner/config.go

```diff
@@ -23,13 +23,23 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// ImperativeTestCaseConfig contains the config only for imperative
+// function run
+type ImperativeTestCaseConfig struct {
+	ExecPath string            `json:"execPath,omitempty" yaml:"execPath,omitempty"`
+	Image    string            `json:"image,omitempty" yaml:"image,omitempty"`
+	Args     map[string]string `json:"args,omitempty" yaml:"args,omitempty"`
+}
+
 // TestCaseConfig contains the config information for the test case
 type TestCaseConfig struct {
-	ExitCode int  `json:"exitCode,omitempty" yaml:"exitCode,omitempty"`
-	Network  bool `json:"network,omitempty" yaml:"network,omitempty"`
-	RunCount int  `json:"runCount,omitempty" yaml:"runCount,omitempty"`
-	Skip     bool `json:"skip,omitempty" yaml:"skip,omitempty"`
-	Debug    bool `json:"debug,omitempty" yaml:"debug,omitempty"`
+	ExitCode         int                      `json:"exitCode,omitempty" yaml:"exitCode,omitempty"`
```

Comment:
Document all these fields, including their default values.

Reply:
Added the comments